.gitignore

```diff
@@ -0,0 +1,13 @@
+_book
```

Comment:
⚠️ not accurate for this repository!

Reply:
Might be copy/paste issue from bonita-doc. Will update it asap